parlai/crowdsourcing/tasks/pairwise_per_turn_eval/task_config/onboard_task_data.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
So this is weird, but I'd probably also make an exact copy of this file called `onboard_task_data__engaging.json`. The reason for this is so that users can see what onboarding data we used for the engagingness task, but also so that we can have the original `onboarding_task_data.json` file around for name compatibility reasons